packages/Takeover/src/components/FocusLock/FocusLock.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FocusLockLibrary from "react-focus-lock";
+import { FocusPropTypes } from "@paprika/helpers/lib/customPropTypes";
+
+const propTypes = FocusPropTypes;
+
+const defaultProps = {
+  returnFocus: true,
+};
+
+const FocusLock = props => <FocusLockLibrary {...props} />;
+
+FocusLock.propTypes = propTypes;
+FocusLock.defaultProps = defaultProps;
+FocusLock.displayName = "Takeover.FocusLock";
+
+export default FocusLock;
```

Comment:
I'm wondering if perhaps this component should be somehow be a shared import or even its own package, but not completely convinced on where it belongs as it is the same component across sidepanel, takeover and modal now

Reply:
Totally fine, duplication is not intrinsically bad, for this scenario I think we are fine.